styles/Home.module.css

```diff
@@ -122,20 +122,46 @@
   }
 }
 
+.pressed-key-portGore {
+  background: #eebbc3 !important;
+}
 
+.currently-pressed-portGore {
+  background: #bae8e8 !important;
+  font-weight: 900;
+}
 
-.pressed-key {
-  background: #abd1c6!important;
-
+.pressed-key-solitare {
+  background: #f582ae !important;
+}
+.currently-pressed-solitare {
+  background: #8bd3dd !important;
+  font-weight: 900;
 }
```

Comment:
Following on my previous comment, is there no way to select the theme colors dynamically using fields from the selected theme interface?
At least that's how I did it for charts(maybe u can refer to my branch to see what I mean). I think it should be possible here too.

Reply:
Hey @aehwany & @reasonablytall , sorry it took so long to get back to you. Unfortunately react-simple-keyboard only accepts CSS class names for the theming (both for the keyboard and it's buttons, so I can't directly work with the theme interface directly 😔. I'll try again and see if I can just interface directly but I don't think it's possible with react-simple-keyboard. Maybe you can take a look and tell me otherwise? 🧐

Reply:
@jtx007 you could create css classes on the parent object with css in js using the themes, and then reference those in any children objects.

Reply:
@reasonablytall should i use styled components to handle the css-in-js or should I use something else?

Reply:
I looks like you can use the styled-jsx provided by next (example [here](https://nextjs.org/docs/basic-features/built-in-css-support#css-in-js)) to define classes in the container of the keyboard.